packages/grafana-ui/src/types/navModel.ts

```diff
@@ -1,22 +1,29 @@
 export interface NavModelItem {
   text: string;
-  url: string;
+  url?: string;
   subTitle?: string;
+  subType?: any;
```

Comment:
can't see this being used anywhere.

Reply:
probably a legacy from old plugin or data source pages. Think the subTitle is all that is actually used, and seems sufficient. Can remove this from here (and PageHeader.tsx)

Reply:
Removed:
```
 {main.subType && (
            <div className="page-header__stamps">
              <i className={main.subType.icon} />
              {main.subType.text}
            </div>
          )}
```

waiting for this to pass CI, then will merge